articles/storage/files/storage-how-to-use-files-linux.md

```diff
@@ -35,10 +35,12 @@ ms.subservice: files
 | Ubuntu Server | 14.04+ | 16.04+ |
 | RHEL | 7+ | 7.5+ |
 | CentOS | 7+ |  7.5+ |
-| Debian | 8+ |   |
+| Debian | 8+ | 9+*, 10+ |
```

Comment:
Thanks for making the earlier change, we appreciate it. :)

However, would you remove Debian 10 from here? We realize that you tested it and it works however, this is a table of the minimum versions. So "9+*" already covers this information. We don't want to enumerate every version in this table. Thanks again!

Reply:
The idea is that 9 needs the back ported kernel and 10 does not - if that is not clear to you, I am happy to remove it. What you think?

Reply:
We'd like to stick with 9+*. Yes it doesn't highlight that 10 doesn't need it but we're fine with that tradeoff in this case. :)

Reply:
done :)